app/domain/authentication/authn_oidc/v2/client.rb

```diff
@@ -135,7 +171,7 @@ def discovery_information(invalidate: false)
             skip_nil: true
           ) do
             @discovery_configuration.discover!(@authenticator.provider_uri)
-          rescue HTTPClient::ConnectTimeoutError, Errno::ETIMEDOUT => e
+          rescue Faraday::TimeoutError, Errno::ETIMEDOUT => e
```

Comment:
Did the `OpenID-Connect` gem move from HTTPClient to Faraday?

Reply:
Yes, only [recently](https://github.com/nov/rack-oauth2/pull/91) in v2.0.0+